bindings/python/src/pgvecto_rs/highapi/client.py

```diff
@@ -0,0 +1,199 @@
+            meta: Mapped[dict] = mapped_column(postgresql.JSONB)
+            embedding: Mapped[ndarray] = mapped_column(Vector(dimension))
+
+        self._engine = create_engine(db_url)
+        with Session(self._engine) as session:
+            session.execute(text("CREATE EXTENSION IF NOT EXISTS vectors"))
+            session.commit()
+        self._table = _Table
+        self._table.__table__.create(self._engine, checkfirst=not new_table)  # type: ignore
+        self.dimension = dimension
+        self.embedder = embedder
+        try:
+            if embedder is not None:
+                if embedder.get_dimension() != dimension:
+                    raise ValueError(
+                        f"Dimension mismatch: (embedder){embedder.get_dimension()} != (given){dimension}"
+                    )
+        except Exception as e:
+            self.drop()
+            raise e
+
+    @classmethod
+    def from_records(
+        cls,
+        records: List[Record],
+        db_url: str,
+        table_name: str,
+        dimension: int,
+        embedder: Optional[BaseEmbbeder] = None,
+    ):
+        client = cls(db_url, table_name, dimension, embedder, True)
+        try:
+            for record in records:
+                client.add_record(record)
+        except Exception as e:
+            client.drop()
+            raise e
+        return client
+
+    @classmethod
+    def from_texts(
+        cls,
+        texts: List[str],
+        meta: Optional[dict],
+        db_url: str,
+        table_name: str,
+        dimension: int,
+        embedder: BaseEmbbeder,
+    ):
+        client = cls(db_url, table_name, dimension, embedder, True)
+        meta = meta or {}
+        try:
+            for i in range(len(texts)):
+                client.add_text(texts[i], meta)
+        except Exception as e:
+            client.drop()
+            raise e
+        return client
+
+    # ================ Insert ================
+    def add_record(self, record: Record) -> None:
+        with Session(self._engine) as session:
+            session.execute(
+                insert(self._table).values(
+                    id=record.id,
+                    text=record.text,
+                    meta=record.meta,
+                    embedding=record.embedding,
+                )
+            )
+            session.commit()
+
+    def add_text(self, text: str, meta: dict = {}) -> Record:
+        if self.embedder is None:
+            raise ValueError("No embedder provided")
+        embedding = self.embedder.embed(text)
+        record = Record.from_text(text, meta, embedding)
+        self.add_record(record)
+        return record
+
+    # ================ Query ================
+
+    def search(
+        self,
+        embedding: Union[ndarray, List[float]],
+        distance_op: Literal["<->", "<=>", "<#>"] = "<->",
+        limit: int = 10,
+        filter: Optional[FilterFunc] = None,
+        order_by_dis: bool = True,
```

Comment:
I think this is 100% True. Are there any requirements to return the results without ordering?